modules/SofaConstraint/SlidingConstraint.inl

```diff
@@ -63,39 +63,40 @@ void SlidingConstraint<DataTypes>::buildConstraintMatrix(const core::ConstraintP
     const Coord B = x2.getValue()[tm2b];
 
     // the axis
-    Coord uniAB = B - A;
-    const Real ab = uniAB.norm();
-    uniAB.normalize();
+
+    Deriv m_dirAxe = B - A;
```

Comment:
This line redefine m_dirAxe which is already defined as a class member.
So the change on this function on m_dirAxe are not stored into the class member but only on this local variable. 

The consequence is that in the line in the store lambda function:
```cpp
 mforce.setValue( m_dirProj* lamb1 + m_dirOrtho * lamb2 + m_dirAxe * lamb3);
```
is not working as expected as the m_dirAxe never change.